File: utils/csharp/input.cs
using System;
using System.IO;

namespace Utils;
public static class Input
{
   public struct Answers
   {
      public string Test1 { get; set; }
      public string Test2 { get; set; }
      public string Part1 { get; set; }
      public string Part2 { get; set; }
   }

   public static string GetInput(string path)
   {
      return File.ReadAllText(path);
   }

   public static Answers GetAnswers(string path)
   {
      File.ReadAllText(path);
      var answers = new Answers();
      answers.Test1 = "";
      answers.Test2 = "";
      answers.Part1 = "";
      answers.Part2 = "";

      return answers;
   }
}

File: utils/java/Input.java
package kival.aoc.utils;

import java.io.IOException;
import java.nio.charset.StandardCharsets;
import java.nio.file.Files;
import java.nio.file.Paths;

public class Input {
   public static class Answers {
      public String test1, test2, part1, part2;
   }

   public static String getInput(String path) throws IOException {
      return new String(Files.readAllBytes(Paths.get(path)), StandardCharsets.UTF_8);
   }

   public static Answers getAnswers(String path) throws IOException {
      String input = getInput(path);
      Answers answers = new Answers();
      int idx = 0;
      for (String line : input.split("\n")) {
         switch (idx) {
            case 0:
               answers.test1 = line;
               break;
            case 1:
               answers.part1 = line;
               break;
            case 2:
               answers.test2 = line;
               break;
            case 3:
               answers.part2 = line;
               break;
         }
         idx++;
      }

      return answers;
   }
}
